index.html

```diff
@@ -259,12 +259,12 @@
             A HTTP <code>POST</code> request that includes an unsupported <code>Digest</code> type (as described
             in [[!RFC3230]]), SHOULD be rejected with a 400 Bad Request response.
           </p>
-          <p>
-            Implementations SHOULD support <code>Content-Type: message/external-body</code> extensions for
-            request bodies for HTTP <code>POST</code> that would create <a>LDP-NR</a>s. This content-type requires
-            a complete <code>Content-Type</code> header that includes the location of the external body, e.g
-            <code>Content-Type: message/external-body; access-type=URL; URL=\"http://www.example.com/file\"</code>,
-            as defined in [[!RFC2017]].
+          <p class='informative'>
+            Implementations may support <code>Content-Type: message/external-body</code> extensions for
+            request bodies for HTTP <code>POST</code> or <code>PUT</code> that would create <a>LDP-NR</a>s.
```

Comment:
Do we need to have both verbs here?  Can we just have only POST in the POST section and PUT in the PUT section?